vscode/src/createProject.ts

```diff
@@ -132,15 +132,18 @@ export async function initProjectCreator(context: vscode.ExtensionContext) {
             },
           );
           for (const [name, type] of dirFiles) {
-            if (type === vscode.FileType.File && name.endsWith(".qs")) {
+            if (
+              type === vscode.FileType.File &&
+              (name.endsWith(".qs") || name.endsWith(".qsc"))
```

Comment:
This is a preexisting bug, but this catches files named `.qs`  and `.qsc`. Maybe explicitly exclude those?

Reply:
done